components/resizable-iframe/index.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Imported from Calypso, with some lint fixes and gutenburg specific changes.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import debugFactory from 'debug';
+import { omit } from 'lodash';
+import uuid from 'uuid/v4';
+
+/**
+ * Globals
+ */
+const debug = debugFactory( 'gutenburg:resizable-iframe' ),
+	noop = () => {};
+
+export default React.createClass( {
+	displayName: 'ResizableIframe',
+
+	propTypes: {
+		src: React.PropTypes.string,
+		width: React.PropTypes.oneOfType( [
+			React.PropTypes.string,
+			React.PropTypes.number
+		] ),
+		height: React.PropTypes.oneOfType( [
+			React.PropTypes.string,
+			React.PropTypes.number
+		] ),
+		onLoad: React.PropTypes.func,
+		onResize: React.PropTypes.func,
+		title: React.PropTypes.string
+	},
+
+	getInitialState: function() {
+		return { width: 0, height: 0 };
+	},
+
+	getDefaultProps: function() {
+		return {
+			onLoad: noop,
+			onResize: noop,
+			title: uuid()
```

Comment:
Even the tweet author doesn't help _that_ much, if for example, you're embedded two tweets by the same author... Perhaps "Embedded tweet from <author>" is a good compromise though? It would be easy enough to parse that out of the url.